MREUnityRuntime/MREUnityRuntime.sln

```diff
@@ -12,22 +12,19 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug_Android|Any CPU = Debug_Android|Any CPU
```

Comment:
Does this mean we're ditching the Debug_Android build target?

Reply:
Eric already removed all cases of debug logging for Android when he brought in Brent's changes.  He did not replace before leaving the project.  I am removing this until we have a decision on a better way to handle debug builds.